src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetDestruction.java

```diff
@@ -260,10 +259,6 @@ public static void anchorBlocks(EntityPlayer player, ItemStack stack) {
 
     public static SortedSet<BlockPos> getArea(World world, BlockPos pos, EnumFacing incomingSide, EntityPlayer player, ItemStack stack) {
         SortedSet<BlockPos> voidPositions = new TreeSet<>(Comparator.comparingInt(Vec3i::getX).thenComparingInt(Vec3i::getY).thenComparingInt(Vec3i::getZ));
-        int depth = getToolValue(stack, "depth");
-        if (depth == 0)
-            return voidPositions;
```

Comment:
**Don't remove shortcuts! ** There's no reason to do anything if the depth is 0.